### ISSUE TITLE
Invalid Syntax?

### ISSUE THREAD

**Issue by Nozu82** [danielchalef/pyethrecoverv3#2]
I'm running Windows with Python 3.6.1 installed. I have zero experience of Python but I've managed to install Python, Bitcoinlib and Joblib. I get the following error after I try to run pyethrecover_v3:

"
pyethrecover_v3.py -w keyfile -f password_spec.txt
  File "C:\Temp\GitRepo\pyethrecoverv3\pyethrecover_v3.py", line 100
    except Exception, e:
                    ^
SyntaxError: invalid syntax
"

Help?

**Comment by Nozu82** [comment]
I found out that the "Exception, e" is part of an older version of Python and new format should be
"Exception as e". After I made these changes the next error follows:

"
pyethrecover_v3.py -w keyfile -f password_spec.txt
Traceback (most recent call last):
  File "C:\Temp\GitRepo\pyethrecover\pyethrecover_v3.py", line 3, in <module>
    from keys import decode_keystore_json #keys.py from pyethereum, we only want the decode_keystore_json function
  File "C:\Temp\GitRepo\pyethrecover\keys.py", line 5, in <module>
    from rlp.utils import decode_hex
ModuleNotFoundError: No module named 'rlp'"

So if I understood correctly a 'rlp' module is missing. Where to find it? Any ideas?
